module/stylesheets/components/_avatar.scss

```diff
@@ -0,0 +1,23 @@
+.avatar {
+  font-size: 32px;
+  display: flex;
+  height: 50px;
+  width: 50px;
+
+  img {
+    height: 50px;
+    width: 50px;
+    border-radius:50px;
+  }
+  span {
+    color: #fff;
+    position: relative;
+    left: 14px;
+    width: 40px;
+    overflow: hidden;
+    text-align: center;
+    margin-left: -8px;
+  }
+  .avatar-rounded {
```

Comment:
I believe this should be `&.avatar-rounded`

Reply:
Fixed